python/ui/setup_project.py

```diff
@@ -650,8 +650,8 @@ def retranslateUi(self, Wizard):
         self.label_2.setText(QtGui.QApplication.translate("Wizard", "Please select either a folder or a zip file.", None, QtGui.QApplication.UnicodeUTF8))
         self.storage_map_page.setTitle(QtGui.QApplication.translate("Wizard", "<p></p><font size=18>&nbsp;Define Storages</font><p></p>", None, QtGui.QApplication.UnicodeUTF8))
         self.storage_map_page.setSubTitle(QtGui.QApplication.translate("Wizard", "&nbsp;", None, QtGui.QApplication.UnicodeUTF8))
-        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://support.shotgunsoftware.com/hc/en-us/articles/219030938-Linking-to-local-files#how_to_set_up\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
-        self.local_file_link_lbl.setText(QtGui.QApplication.translate("Wizard", "Linking to local files must be enabled in your <a href=\"https://support.shotgunsoftware.com/hc/en-us/articles/219030938\">ShotGrid Preferences</a>", None, QtGui.QApplication.UnicodeUTF8))
+        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://help.autodesk.com/view/SGSUB/ENU/?guid#how-do-i-set-up-local-file-storage-for-networked-resources-such-as-a-shared-element-library-\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
```

Comment:
I think https://help.autodesk.com/view/SGSUB/ENU/?guid#how-do-i-set-up-local-file-storage-for-networked-resources-such-as-a-shared-element-library-\
should be https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html